CHANGELOG

```diff
@@ -1,3 +1,17 @@
+snowplow-media-player 0.7.2 (2024-03-18)
```

Comment:
I'm sure there are reasons but it is a week ahead.

Reply:
I gotta do most of the packages so giving myself time to get them all ready for the same day :)